posthog/settings/data_stores.py

```diff
@@ -1,4 +1,5 @@
 import os
+from typing import Dict, Any
```

Comment:
nit: In Python 3.10 you can just use `dict`.

Reply:
oh lovely! thanks 🤩